tls-proxy-demo/compose.yaml

```diff
@@ -2,7 +2,7 @@ services:
   httpd:
     image: httpd-mpc-ssl
     ports:
-      - "8000:8000"
+      - "8090:8090"
```

Comment:
We should expose 2 separate ports. One for HTTP and another for MCMP.

Reply:
Done.